capstone/src/main/java/com/google/sps/servlets/PrototypeStudents.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+/** Hard-coded student data for prototype */
+public final class PrototypeStudents {
+  public static final String MEGAN_EMAIL = "meghashi@google.com";
+  public static final String MEGHA_EMAIL = "kakm@google.com";
+  public static final String KEVIN_EMAIL = "kshao@google.com";
+  public static final String MEGAN_NAME = "Megan Shi";
+  public static final String MEGHA_NAME = "Megha Kak";
+  public static final String KEVIN_NAME = "Kevin Shao";
+  public static final Integer YEAR_2022 = 2022;
```

Comment:
Better is to just use int, right?

Reply:
Same with below - I think the primitive type is preferred for some reason. I could be wrong, though?

Reply:
I think you're right! Just changed it